radixdlt-core/docker/scripts/generate-yml.sh

```diff
@@ -53,7 +104,7 @@ do
     echo "    networks:">>${file_name}
     echo "      - $network_name">>${file_name}
     echo "    ports:">>${file_name}
-    echo "      - \"80${p0}:8080\"">>${file_name}
+    echo "      - \"${port_param}${p0}:8080\"">>${file_name}
```

Comment:
Suggest that this would be more predictable if `port_param` was the initial port number, and that subsequent nodes used subsequent port numbers, rather than dividing things into two parts here.

For example, it should be the case that using `-p 8080 -n 2` would result in ports 8080 and 8081 being generated.  In order to do this at the moment, would need to use `-p 80 -n 2` which is kind of confusing.  Also, it is not currently possible to generate (for example) configs that use ports 8000, 8001 etc, given the way that `p0` is set up here now.

Reply:
Yep, good point.